packages/hoprd/hopr-admin/src/commands/info.ts

```diff
@@ -0,0 +1,40 @@
+import type API from '../utils/api'
+import { toPaddedString } from '../utils'
+import { Command, type CacheFunctions } from '../utils/command'
+
+export default class Info extends Command {
+  constructor(api: API, cache: CacheFunctions) {
+    super(
+      {
+        default: [[], 'displays node information']
+      },
+      api,
+      cache
+    )
+  }
+
+  public name() {
+    return 'info'
+  }
+
+  public description() {
+    return 'Information about the HOPR Node, including any options it started with'
+  }
+
+  public async execute(log: (msg: string) => void, _query: string): Promise<void> {
+    const nodeInfoRes = await this.api.getInfo()
+    if (!nodeInfoRes.ok) return log(this.invalidResponse('get node information'))
+    const nodeInfo = await nodeInfoRes.json()
+
+    return log(
+      toPaddedString([
```

Comment:
Regression: missing NR related information and Connectivity Health Indicator.
See `hoprd/src/commands/info.ts` from line 30:
````
 `HOPR NetworkRegistry: ${smartContractInfo.hoprNetworkRegistryAddress}`,
        `Eligibility: ${await this.node.isAllowedAccessToNetwork(this.node.getId())}`,
        `Connectivity Indicator: ${this.node.getConnectivityHealth().toString()}`,
        `Channel closure period: ${channelClosureMins} minutes`
````

Reply:
I fixed this.